### PR TITLE
Replace broken relative links in documentation with working full URLs.   See:  Issue #270

### PR DESCRIPTION
**Pull request by andersje**
as i was trying to set up promscale and the extensions, this is one instance i found of issue #270 -- I'm pretty sure i got the links right, can someone validate?

### CODE REVIEW

**Review by cevian — APPROVED** [review]
Thanks @andersje, this looks great

**Comment by antekresic** [comment]
Would you mind squashing the commits so we can get this merged?

File: docs/binary.md [hunk 1]
```diff
@@ -5,10 +5,10 @@
 You can download pre-built binaries for the Promscale
 Connector [on our release page](/releases).
```
Comment (antekresic) [inline docs/binary.md:6]: This link needs to be changed to https://github.com/timescale/promscale/releases as well.

**Review by antekresic — CHANGES_REQUESTED** [review]
Would you mind updating the releases page link as well? And also squashing the commits into one so we can get this merged?

Thanks!

File: docs/binary.md (hunk 1 above)
Comment (andersje) [reply docs/binary.md]: done!  :)

**Comment by cevian** [comment]
Thanks @andersje

**Pull request merged**